Objects/codeobject.c

```diff
@@ -1906,3 +1906,18 @@ _PyCode_ConstantKey(PyObject *op)
     }
     return key;
 }
+
+void 
+_PyStaticCode_Dealloc(PyCodeObject *co, _Py_CODEUNIT *firstinstr)
+{
+    PyMem_Free(co->co_quickened);
```

Comment:
The code in `code_dealloc` is:
```C
    if (co->co_quickened) {
        PyMem_Free(co->co_quickened);
        _Py_QuickenedCount--;
    }
```
Why does this differ?

Reply:
@kumaraditya303, Mark was asking why you don't decrement `_Py_QuickenedCount` here, since it is decremented whenever a code object is deallocated. After `Py_Finalize` this count should be back to zero, unless code objects are being leaked somehow.

Reply:
Yes upon reading the specialize.c _Py_QuickenedCount is supposed to be decremented when a codeobject upon deallocation, (I thought that it is some sort of stat used for some profiling etc so it would be required when the process it about to exit but it is different.)